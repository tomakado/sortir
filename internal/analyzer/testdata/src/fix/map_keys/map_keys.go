package map_keys

var m1 = map[string]int{
	"zebra": 3,
	"apple": 1, // want "composite literal elements are not sorted"
	"banana": 2,
}

func foo() {
	m2 := map[string]string{
		"z": "Z",
		"a": "A", // want "composite literal elements are not sorted"
		"b": "B",
	}
	_ = m2
}