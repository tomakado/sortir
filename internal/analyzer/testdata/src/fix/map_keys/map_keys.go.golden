package map_keys

var m1 = map[string]int{
	"apple": 1,
	"banana": 2,
	"zebra": 3,
}

func foo() {
	m2 := map[string]string{
		"a": "A",
		"b": "B",
		"z": "Z",
	}
	_ = m2
}